{{cookiecutter.repo_name}}/.pii_annotations.yml

```diff
@@ -0,0 +1,15 @@
+source_path: ./
+report_path: pii_reports
+safelist_path: .annotation_safe_list.yml
+coverage_target: 100.0
+annotations:
+    ".. no_pii::":
+    "pii_group":
+        - ".. pii::":
+        - ".. pii_types::":
+            choices: [id, name, other]
```

Comment:
What do we think about the long list of pii types in OEP-30 under "The potential values of pii_types are:"?  Should we populate all those values here too?

Reply:
Yes - I'll do this.